src/main/java/com/teamchallenge/bookti/review/UserReviewServiceImp.java

```diff
@@ -0,0 +1,139 @@
+package com.teamchallenge.bookti.review;
+
+import com.teamchallenge.bookti.exception.user.UserNotFoundException;
+import com.teamchallenge.bookti.user.User;
+import com.teamchallenge.bookti.user.UserRepository;
+import com.teamchallenge.bookti.user.dto.UserProfileDTO;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.List;
+import java.util.Objects;
+
+import static java.text.MessageFormat.format;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class UserReviewServiceImp implements UserReviewService {
+
+    private final UserRepository userRepository;
+    private final UserReviewRepository userReviewRepository;
+    private final JwtDecoder jwtDecoder;
+
+    @Override
+    public List<UserReview> findAllUserReceivedReviewsById(Integer userId) {
+        if (userId > 0) {
+            List<UserReview> userReviewList = userReviewRepository.findAllUserReceivedReviewsById(userId);
+            log.info("From UserReviewServiceImp method - findAllUserReviewById - return list or empty list, received review to user: {}.", userId);
+
+            return userReviewList;
+        } else {
+            throw new UserNotFoundException("Write correct Users id.");
+        }
+    }
```

Comment:
```suggestion
    @Override
    public List<UserReview> findAllUserReceivedReviewsById(Integer userId) {
      if (!userRepository.existsById(userId)) {
        log.info("message");
        throw new UserNotFoundException(String.format(UserConstant.NOT_FOUND_MESSAGE, userId));        
      }
      List<UserReview> userReviewList = userReviewRepository.findAllUserReceivedReviewsById(userId);
      log.info("From UserReviewServiceImp method - findAllUserReviewById - return list or empty list, received review to user: {}.", userId);
      return userReviewList;
    }
```
If you need only check the existence of the model without reading the data -> use existsBy();
If you need to read model data and verify that the model is present -> use findBy();